### PR TITLE
2020 Amendments

### PR DESCRIPTION
**Pull request by noahzemlin**

### CODE REVIEW

File: constitution/constitution.tex [hunk 1]
```diff
@@ -32,13 +32,19 @@ \section{}
...
 \section{}
-The Officers of the Organization shall be elected by a simple majority of the members attending the election meeting who have served on an SCR team.
+The Voting Membership shall be defined as any member who is an active Officer, a Team Captain, or a Team Lead. SCR members may request to the Officers to become Voting Membership until the end of the academic year.
 
 \section{}
-Any Officer or Advisor may be removed by a three-fourths vote of the qualified voting membership. 
+The Officers of the Organization shall be elected by the Voting Membership.
+
+\section{}
+Any Officer or Advisor may be removed by a two-thirds votes of the Officers.
```
Comment (kevin-robb) [inline constitution/constitution.tex]:
```suggestion
Any Officer or Adviser may be removed by a two-thirds vote of the Officers.
```

File: constitution/constitution.tex [hunk 2]
```diff
@@ -32,13 +32,19 @@ \section{}
...
+
+\section{}
+Any Officer or Advisor may be removed by a three-fourths vote of those in attendance at an Officer Impeachment meeting. A petition with at least one-third of the Voting Membership shall result in an Officer Impeachment meeting within two weeks of when the petition is brought to the President or Vice President. The meeting time shall be voted on by the Voting Membership and the Officers.
```
Comment (kevin-robb) [inline constitution/constitution.tex]:
```suggestion
Any Officer or Advisor may be removed by a three-fourths vote of those in attendance at an Officer Impeachment meeting. A petition signed by at least one-third of the Voting Membership shall result in an Officer Impeachment meeting within two weeks of when the petition is brought to the President or Vice President. The meeting time shall be decided by the Voting Membership and the Officers.
```

File: constitution/constitution.tex [hunk 3]
```diff
@@ -32,13 +32,19 @@ \section{}
...
 
 \section{}
 The President shall be the chief executive officer of the Organization.  The President shall preside at all meetings and direct the affairs of the Organization with the advice and consent of the other officers.  The President shall decide what competition teams may be a part of the Organization based on the Organization’s current capabilities. The President shall ensure that the goals of leadership and success are met by all. In the event that there is no current President, the remainder of the President’s term shall be filled in the following order of succession:  Vice-President, then Treasurer, then Secretary, then a qualified student of the Advisor’s choice.
```
Comment (kevin-robb) [inline constitution/constitution.tex]: I still hold that Secretary should be ranked above Treasurer, here and in all other places officers are listed or put in hierarchy.

File: constitution/constitution.tex [hunk 4]
```diff
@@ -53,7 +59,13 @@ \section{}
 The Secretary shall maintain minutes of each meeting and shall document every election. The Secretary is responsible for maintaining a list of qualified members of the Organization.
```
Comment (kevin-robb) [inline constitution/constitution.tex]:
```suggestion
The Secretary shall maintain minutes of each meeting and shall document every election. The Secretary is responsible for maintaining a list of qualified members of the Organization by keeping the team roster up to date, and adding graduates' information to the alumni roster.
```

File: constitution/constitution.tex [hunk 5]
```diff
@@ -53,7 +59,13 @@ \section{}
...
 
 \section{}
-The Organization shall have as an Advisor a fulltime member of the University faculty or staff.
+The Social Media shall .
```
Comment (kevin-robb) [inline constitution/constitution.tex]:
```suggestion
The Social Media officer shall be responsible for taking photos at any events and at some meetings of each team, and posting pictures and updates to any of our social media platforms. These include Facebook, Instagram, Twitter, and LinkedIn. 
```

File: constitution/constitution.tex [hunk 6]
```diff
@@ -53,7 +59,13 @@ \section{}
...
+
+\section{}
+The Lab Manager shall .
```
Comment (kevin-robb) [inline constitution/constitution.tex]:
```suggestion
The Lab Manager shall be responsible for restocking lab supplies such as those on the common parts list, keeping the lab clean and organized, and making large-scale end-of-year purchases with extra funds with the approval of the President.
```

File: constitution/constitution.tex [hunk 7]
```diff
@@ -73,16 +85,22 @@ \section{}
...
 \section{}
-When an election is for more than one office, the order of voting shall be in the following order: President, then Vice-President, then Treasurer, then Secretary.
+When an election is for more than one office, the order of voting shall be in the following order: President, then Vice-President, then Treasurer, then Secretary, then Social Media, then Lab Manager.
 
 \section{}
-Any qualified member of the Organization may run for any office. 
+Any qualified member of the Organization may run for any office.
 
 \section{}
 A qualified member may be a candidate for more than one office, but shall immediately remove himself or herself for consideration of any other Officer position after being elected. In no instance may one person fulfill more than one Officer position.
```
Comment (kevin-robb) [inline constitution/constitution.tex]:
```suggestion
A qualified member may be a candidate for more than one office, but shall be immediately removed from consideration of any other Officer position after being elected. In no instance may one person fulfill more than one Officer position. 
```

Comment (kevin-robb) [reply constitution/constitution.tex]: gotta make it gender neutral!

File: constitution/constitution.tex [hunk 8]
```diff
@@ -93,7 +111,7 @@ \section{}
 The Organization is  subject to Local, State, and Federal Laws.
```
Comment (kevin-robb) [inline constitution/constitution.tex]:
```suggestion
The Organization is subject to Local, State, and Federal Laws.
```

File: constitution/constitution.tex [hunk 9]
```diff
@@ -93,7 +111,7 @@ \section{}
...
 
 \section{}
-The Organization is indepent of any national or
+The Organization is independent of any national or
 parent organization.
```
Comment (kevin-robb) [inline constitution/constitution.tex]:
```suggestion
The Organization is independent of any national or parent organization.
```

**Review by kevin-robb — COMMENTED** [review]
I added my comments, mostly changes to wording/grammar. Added social media and lab manager roles. I still think Secretary should be ranked above Treasurer.

**Review by SarahBrown — APPROVED** [review]
Approved by all officers present, 5/6 vote.

**Pull request merged**